core/templates/layout.html

```diff
@@ -24,7 +24,7 @@
 <a class="lpanel check-in" href=/checked-out data-title="Check In">Check In</a>
 <a class="lpanel">Edit Catalog</a>
 <a class="lpanel" href="https://hummedia.byu.edu/api/v2//account/login?r=https://hummedia.byu.edu">Hummedia</a>
-<a class="lpanel" data-title=Messages href="/message?">Messaging</a>
```

Comment:
Hahaha--This actually appends a question mark to all of the links, so even
though it's not in the HTML, the JavaScript takes care of it by appending
?ajax=true
On Mar 18, 2015 8:17 AM, "trevren11" <notifications@github.com> wrote:

> In core/templates/layout.html
> <https://github.com/BYU-ODH/hlrdesk/pull/190#discussion_r26666118>:
>
> > @@ -24,7 +24,7 @@
> >  <a class="lpanel check-in" href=/checked-out data-title="Check In">Check In</a>
> >  <a class="lpanel">Edit Catalog</a>
> >  <a class="lpanel" href="https://hummedia.byu.edu/api/v2//account/login?r=https://hummedia.byu.edu">Hummedia</a>
> > -<a class="lpanel" data-title=Messages href="/message?">Messaging</a>
>
> I need that question mark!! haha I have changed that several times because
> it keeps getting deleted.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/190/files#r26666118>.
>